README.md

```diff
@@ -5,7 +5,11 @@
 [![TypeScript](https://img.shields.io/badge/%3C%2F%3E-TypeScript-blue.svg)](https://www.typescriptlang.org/)
 [![ES Modules](https://img.shields.io/badge/ES-Modules-brightgreen)](https://github.com/Chatie/tsconfig/issues/16)
 
-## About Wechaty
```

Comment:
Could you please explain why the `About Wechaty` should be replaced by `Wechaty`?

I can not understand it.